cms/templates/cms/wizards/base.html

```diff
@@ -15,6 +15,7 @@
 {% block extrahead %}
     {{ block.super }}
     <script src="{% static "cms/js/dist/bundle.admin.base.min.js" %}" type="text/javascript"></script>
+    {{ form.media }}
     <script>
     $(function () {
```

Comment:
This JS file is part of a specific "widget" for a specific field for a specific cms_wizard. It doesn't make sense to combine it with anything else. This should not be included on all cms_wizard forms.